mappings/net/minecraft/world/WorldServer.mapping

```diff
@@ -38,11 +61,21 @@ CLASS none/lu net/minecraft/world/WorldServer
 	METHOD am shouldSpawnAnimals ()Z
 	METHOD an initDebugWorldProperties ()V
 	METHOD b ()Lnone/aiw;
+	METHOD b hasBlockTickScheduled (Lnone/cn;Lnone/aky;)Z
```

Comment:
This method takes a BlockPos and Block / IBlockState, so to me it reads best as "does this BlockPos/Block pair have a block tick scheduled?".